src/sage/rings/number_field/number_field_rel.py

```diff
@@ -2467,7 +2467,7 @@ def order(self, *gens, **kwds):
         EXAMPLES::
 
             sage: P.<a,b,c> = QQ[2^(1/2), 2^(1/3), 3^(1/2)]                             # needs sage.symbolic
-            sage: R = P.order([a,b,c]); R                                               # needs sage.symbolic
+            sage: R = P.order([a,b,c]); R                                               # long time (83s), needs sage.symbolic
             Relative Order generated by
```

Comment:
This change causes the doctest dataflow warning below

Reply:
You can fix it by add `# long time` to line 2481

Reply:
This command also consumes ~2G of memory.

Reply:
@gmou3 Is it possible to annotate that?

Reply:
I was pointing it out in case someone wants to change this test. I am not aware of a standardized way to annotate excessive memory usage.

Reply:
Probably we would not want to run this test automatically -- both because 83 s is too long even for a `# long time` test and because of the memory use.